### PR TITLE
Fixednormalization

### LINKED ISSUE #2: Normalize based only on training data
At the moment the mean and standard deviation are calculated over the entire training and validation dataset. Should only be calculated based on training set. This necessarily requires us to have a fixed training and validation set, or to normalize at runtime. Fixed training/test sets are better for repeatability however.

### PR DESCRIPTION
**Pull request by mansdell**
I've made the changes that fix #2 and I've tested it--gives basically the same result as the pre-normalized dataset, but now with a mean/std that is calculated only over a fixed training set. 

- switched over to normalizing data during training (need to use new dataset! can find on server at /data/ao_phase/phase_screens_part1_raw)
- updated mean and std values for normalization, now calculated over just fixed training set
- fixed train-val split (first 80% vs. last 20% of phase_screens_part1)
- fixed bug with tqdm usage to enable testing (i.e., no output) again
- minor changes to clean up format

### CODE REVIEW

**Review by yanii — APPROVED** [review]
Looks good to me
